### PR TITLE
[tests] Add tests for Sensor API

### PR DESCRIPTION
**Pull request by cuiyanx**
Result:

Num | pass | total
---- | ---- | ----
single | 12 | 14
all | 36 | 42

Issue: [#595](https://github.com/01org/zephyr.js/issues/595)
Issue: [#596](https://github.com/01org/zephyr.js/issues/596)

Signed-off-by: Cui, Yan yanx.cui@intel.com

### CODE REVIEW

**Comment by haoxli** [comment]
@cuiyanx  please add UT result.

File: modules/GenericSensor.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2016, Intel Corporation.
```
Comment (grgustaf) [inline modules/GenericSensor.js]: Please update copyrights. 2016-2017 for this one that reuses some old code. 2017 for other files that are all new.

File: modules/GenericSensor.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+// JavaScript library for the tests sensor case
+
+function GenericSensor () {
```
Comment (grgustaf) [inline modules/GenericSensor.js]: Follow our style by removing space before (), also line 13, 32, 56, 63, 96, 153, etc.

**Comment by grgustaf** [comment]
@brianjjones could you review this one since you've touched sensors before. :)

**Comment by brianjjones** [comment]
LGTM +1

**Comment by grgustaf** [comment]
+1

**Pull request merged**